be/src/olap/aggregate_func.h

```diff
@@ -408,7 +408,11 @@ struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_HLL_UNION, OLAP_FIELD_TYPE_HLL
         dst_slice->size = sizeof(HyperLogLog);
         // use 'placement new' to allocate HyperLogLog on arena, so that we can control the memory usage.
         char* mem = arena->Allocate(dst_slice->size);
-        dst_slice->data = (char*) new (mem) HyperLogLog(src_slice->data);
+        if (src_slice->empty()) {
+            dst_slice->data = (char*) new (mem) HyperLogLog();
```

Comment:
When the slice size is empty?

Reply:
When HLL columns are NULL in the LOAD data